manifests/spoke/spoke-registration-clusterrole.yaml

```diff
@@ -0,0 +1,13 @@
+# Clusterrole for work agent in addition to admin clusterrole.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:open-cluster-management:{{ .SpokeCoreName }}-registration-agent
+rules:
+# Allow agent to get/list/watch nodes.
+- apiGroups: [""]
+  resources: ["nodes", "configmaps", "secrets"]
```

Comment:
What do we see all secrets for?  Do we just need this on one namespace?

Reply:
if we just need it on one namespace, then we're looking at having a clusterrole for nodes and SARs and a role for configmaps and secrets

Reply:
in registration agent, the informer currently watch secret of all namespaces. Need to update registration at first.